README.rst

```diff
@@ -67,11 +67,11 @@ The download includes cardano-node.exe and a .dll. To run the node with cardano-
 Docker image
 ============
 
-You can pull the docker image with the latest version of cardano-node from `here <https://hub.docker.com/r/inputoutput/cardano-node>`_.
+You can pull the docker image with the latest version of cardano-node from `here <https://github.com/IntersectMBO/cardano-node/pkgs/container/cardano-node>`_.
 
 .. code-block:: console
 
-    docker pull inputoutput/cardano-node
+    docker pull ghcr.io/intersectmbo/cardano-node
```

Comment:
This command didn't work for me:
```
% podman pull ghcr.io/intersectmbo/cardano-node
Trying to pull ghcr.io/intersectmbo/cardano-node:latest...
Error: initializing source docker://ghcr.io/intersectmbo/cardano-node:latest: reading manifest latest in ghcr.io/intersectmbo/cardano-node: manifest unknown
```
But this works:
```
podman pull ghcr.io/intersectmbo/cardano-node:8.7.2
```


probably because we don't have tag `latest`, which is used implicitly. Can you add a tag here, or add an information to append a tag to the command?

Reply:
That's important to mention. I'm happy to amend the PR.